### ISSUE TITLE
Switch opticks submodule to bitbucket repository

### ISSUE THREAD

**Issue by plexoos** [BNLNPPS/esi-shell#40]
opticks is actively developed using a Git repo hosted on bitbucket.org

**Comment by plexoos** [comment]
```
#1 0.274 Submodule 'opticks' (git@bitbucket.org:simoncblyth/opticks.git) registered for path 'opticks'
#1 0.275 Cloning into '/var/lib/buildkit/runc-overlayfs/snapshots/snapshots/2/fs/opticks'...
#1 0.364 Warning: Permanently added 'bitbucket.org' (ED25519) to the list of known hosts.
#1 0.395 git@bitbucket.org: Permission denied (publickey).
#1 0.395 fatal: Could not read from remote repository.
#1 0.395 
#1 0.395 Please make sure you have the correct access rights
#1 0.395 and the repository exists.
#1 0.395 fatal: clone of 'git@bitbucket.org:simoncblyth/opticks.git' into submodule path '/var/lib/buildkit/runc-overlayfs/snapshots/snapshots/2/fs/opticks' failed
#1 0.395 Failed to clone 'opticks'. Retry scheduled
#1 0.397 Cloning into '/var/lib/buildkit/runc-overlayfs/snapshots/snapshots/2/fs/opticks'...
#1 0.474 git@bitbucket.org: Permission denied (publickey).
#1 0.474 fatal: Could not read from remote repository.
#1 0.474 
#1 0.474 Please make sure you have the correct access rights
#1 0.474 and the repository exists.
#1 0.474 fatal: clone of 'git@bitbucket.org:simoncblyth/opticks.git' into submodule path '/var/lib/buildkit/runc-overlayfs/snapshots/snapshots/2/fs/opticks' failed
#1 0.474 Failed to clone 'opticks' a second time, aborting
#1 ERROR: failed to update submodules for https://github.com/BNLNPPS/esi-opticks.git: git error: exit status 1
```

**Comment by plexoos** [comment]
Resolved in #39
